editor/gui/editor_file_dialog.cpp

```diff
@@ -207,6 +207,10 @@ void EditorFileDialog::_update_theme_item_cache() {
 
 void EditorFileDialog::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_POSTINITIALIZE: {
+			set_translation_domain(SNAME("godot.editor"));
+		} break;
```

Comment:
We should probably do the same for `EditorSpinSlider`?
I didn't check if it has translatable strings but I assume maybe some tooltips at least.

Reply:
`EditorSpinSlider` currently does not use auto translation. Text translations, including tooltip translations, are done via `TTR` macros.